compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/ComposeAccessible.kt

```diff
@@ -78,8 +80,17 @@ internal class ComposeAccessible(
     var semanticsNode: SemanticsNode,
     val controller: AccessibilityControllerImpl? = null
 ) : Accessible {
+    private val isNativelyInitialized = AtomicBoolean(false)
+
     val accessibleContext: ComposeAccessibleComponent by lazy { ComposeAccessibleComponent() }
-    override fun getAccessibleContext(): AccessibleContext = accessibleContext
+
+    override fun getAccessibleContext(): AccessibleContext {
```

Comment:
> Test: a11y can be tested only manually for now

Could you add a reproducer of the issue to the description? So everyone can check and understand the fix in the future.

Reply:
Sure

Reply:
Maybe even add it in the comment itself

Reply:
Worth to add to the desc as well